modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -63,7 +63,7 @@ TIMEOUT 20
 PROMPT 0
 LABEL pxeboot
     KERNEL http://<HTTP_server>/rhcos-<version>-<architecture>-installer-kernel <1>
-    APPEND ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-<architecture>-installer-initramfs.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal-bios.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <2> <3>
+    APPEND ip=dhcp rd.neednet=1 initrd=rhcos-<version>-<architecture>-installer-initramfs.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal-bios.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <2> <3>
```

Comment:
I think this change should be at line 81 instead, unless it applies to both iPXE and PXE.

@morenod, will you PTAL?

Reply:
@morenod Thanks. I'm looking into your comment in the BZ.

Reply:
@rvanderp3 See the above comment. Per the BZ, could the problem be that http:\\the-server:8080\rhcos-4.1.0-x86_64-installer-initramfs.img should be http:// instead of http:\\?

Reply:
@morenod Please see https://bugzilla.redhat.com/show_bug.cgi?id=1741922#c6 and advise. Thank you!